client/lib/analytics/statsd.js

```diff
@@ -1,35 +0,0 @@
-import config from '@automattic/calypso-config';
-import debug from 'debug';
-import { statsdTimingUrl, statsdCountingUrl } from 'calypso/lib/analytics/statsd-utils';
```

Comment:
Looking at remaining usages of the `statsd-utils` module, could we also deprecate the `logSectionResponse` middleware? It seems like something that was made obsolete by the RUM framework (cc @scinos)